src/changelog.ts

```diff
@@ -282,4 +276,63 @@ export default class Changelog {
       })
     );
   }
+
+  private createReleaseNotesSection(
+    commits: IExtendedCommit[],
+    sections: string[]
+  ) {
+    if (!commits.length) {
+      return;
+    }
+
+    const visited = new Set<number>();
+    let section = '';
+
+    commits.map(commit => {
+      const pr = commit.pullRequest;
+
+      if (!pr || !pr.body) {
+        return;
+      }
+
+      const title = /^> [#]{0,5} Release Notes$/;
```

Comment:
This match is pretty strict. I think it'd be worth loosening the whitespace requirements and making it case insensitive.